### PR TITLE
Add log message when lookup key does not exist.

### PR DESCRIPTION
**Pull request by marcinjzalewski**

### CODE REVIEW

**Comment by marcinjzalewski** [comment]
After the fix it will log lookup key error to the prompt: 
```
[2022-02-16 13:52:29,970] [ERROR] [57285] [phiera.phiera] [phiera.py/get_key:315]: Lookup key: 'module.my_module.client_secret' not found. Make sure you are providing this value in yaml configuration.
```

File: phiera/phiera.py [hunk 1]
```diff
@@ -304,8 +308,10 @@ def get_key(self, key, paths, context, merge):
 
         if merge and key in merges and merges[key].value is not None:
             return merges[key].value
-
-        raise KeyError(key)
+        else:
```
[the remark this answers is not in the document]
Comment (marcinjzalewski) [reply phiera/phiera.py:311]: Done.

File: phiera/phiera.py [hunk 2]
```diff
@@ -9,6 +11,8 @@
 interpolate = re.compile(r'''%\{(?:::|)([^\}]*)\}''')
 rformat = re.compile(r'''%{(?:::|)([a-zA-Z_-|\d]+)}''')
 
+log = logging.getLogger(__name__)
```
Comment (manasib) [inline phiera/phiera.py:14]: can we pick one format for the logger? we can either call it log or LOGGER

**Review by manasib — APPROVED** [review]
LGTM

**Pull request merged**